src/mixpanelMiddleware.js

```diff
@@ -46,6 +47,10 @@ export default function mixpanelMiddleware(token, options = {}) {
             });
         }
 
+        if (timeEvent && timeEvent.length > 0) {
+            mixpanel.time_event(timeEvent);
```

Comment:
Ah ok, sorry I think I must have been looking at something different at the time. 👍